tastyworks/models/position.py

```diff
@@ -0,0 +1,127 @@
+from datetime import date, datetime
+from decimal import Decimal
+from tastyworks.models.alert import Alert, AlertField, Operator
+from tastyworks.models.order import (Order, OrderDetails, OrderPriceEffect, OrderType)
+from tastyworks.models.option import Option, OptionType
+from tastyworks.models.underlying import UnderlyingType
+from enum import Enum
+from dataclasses import dataclass
+
+
+class PositionCostEffect(Enum):
+    CREDIT = 'Credit'
+    DEBIT = 'Debit'
+    NONE = 'None'
+
+
+class InstrumentType(Enum):
+    EQUITY_OPTION = 'Equity Option'
+    NONE = None
+
+
+class QuantityDirection(Enum):
+    LONG = 'Long'
+    SHORT = 'Short'
+    NONE = None
+
+
+@dataclass
+class Position(object):
+    account_number: str = None
+    symbol: str = None
+    instrument_type: InstrumentType = InstrumentType.NONE
+    underlying_symbol: str = None
+    quantity: int = None
+    quantity_direction: QuantityDirection = QuantityDirection.NONE
+    close_price: Decimal = None
+    average_open_price: Decimal = None
+    average_yearly_market_close_price: Decimal = None
+    mark: Decimal = None
+    mark_price: Decimal = None
+    multiplier: int = 100
+    cost_effect: PositionCostEffect = PositionCostEffect.NONE
+    is_suppressed: bool = None
+    is_frozen: bool = None
+    restricted_quantity: int = 0
+    realized_day_gain: Decimal = None
+    realized_day_gain_effect: PositionCostEffect = PositionCostEffect.NONE
+    realized_day_gain_date: date = None
+    created_at: datetime = None
+    updated_at: datetime = None
+
+    def is_option(self):
+        if self.instrument_type == InstrumentType.EQUITY_OPTION:
+            return True
+        else:
+            return False
+
+    def get_option_obj(self):
+        exp_date = datetime.strptime(self.symbol[6:12], '%y%m%d').date()
+        option_type = OptionType(self.symbol[12:13])
+        strike = Decimal(self.symbol[13:]) / 1000
+        return Option(ticker=self.underlying_symbol, quantity=self.quantity, expiry=exp_date, strike=strike, option_type=option_type, underlying_type=UnderlyingType.EQUITY)
+
+    def get_closing_order_price_effect(self):
+        if self.cost_effect == PositionCostEffect.CREDIT:
+            return OrderPriceEffect.CREDIT
+        elif self.cost_effect == PositionCostEffect.DEBIT:
+            return OrderPriceEffect.DEBIT
+
+    def get_closing_order_object(self, price: Decimal, stop_trigger: Decimal = None, order_type: OrderType = OrderType.LIMIT):
+        closing_order_price_effect = self.get_closing_order_price_effect()
```

Comment:
given that this whole method deals with creating a closing order, could this not just be called `price_effect`?